dff/context_storages/__init__.py

```diff
@@ -3,11 +3,18 @@
 from .db_connector import DBAbstractConnector, DBConnector, threadsafe_method  # noqa: F401
 from .json_connector import JSONConnector  # noqa: F401
 from .pickle_connector import PickleConnector  # noqa: F401
-from .sql_connector import SQLConnector  # noqa: F401
-from .ydb_connector import YDBConnector  # noqa: F401
-from .redis_connector import RedisConnector  # noqa: F401
-from .mongo_connector import MongoConnector  # noqa: F401
-from .protocol import PROTOCOLS
+from .sql_connector import (  # noqa: F401
+    SQLConnector,  # noqa: F401
+    postgres_available,  # noqa: F401
+    mysql_available,  # noqa: F401
+    sqlite_available,  # noqa: F401
+    sqlalchemy_available,  # noqa: F401
+)  # noqa: F401
+from .ydb_connector import YDBConnector, ydb_available  # noqa: F401
```

Comment:
After #46 merging:
- [ ] mv 'redis_connector' to 'redis' and so on
- [ ] rename RedisConnector to RedisContextStorage

Reply:
Structure of tests was already updated, wasn't it?

Reply:
Sorry, I reviewer from a phone and missed it